src/pkg/caendr/caendr/services/sql/dataset/_env.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from caendr.utils.data import remove_env_escape_chars
+
+
+# Get environment variables
+MODULE_DB_OPERATIONS_BUCKET_NAME = os.environ.get('MODULE_DB_OPERATIONS_BUCKET_NAME')
+STRAIN_VARIANT_ANNOTATION_PATH   = os.environ.get('STRAIN_VARIANT_ANNOTATION_PATH')
+
```

Comment:
add guards here -- raise exception if neither of these exists

Reply:
Started working on better guards across the codebase in new branch: https://github.com/AndersenLab/CAENDR/tree/enhancement/get-env-vars